src/components/HURUmap/KeyMetric/index.js

```diff
@@ -0,0 +1,58 @@
+import { Typography, LinearProgress } from "@material-ui/core";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const KeyMetric = ({ formattedValue, value, title, color, description }) => {
+  const classes = useStyles();
+
+  if (!(formattedValue && title)) {
+    return null;
+  }
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.dataSection}>
+        <Typography variant="h3">{formattedValue || value}</Typography>
+        <Typography
+          variant="caption"
+          className={clsx(classes.text, classes.title)}
+        >
+          {title}
+        </Typography>
+        <LinearProgress
+          className={classes.progressBar}
+          value={value}
+          color={color}
+          variant="buffer"
+        />
+      </div>
+      <Typography
+        variant="caption"
+        className={clsx(classes.description, classes.text)}
+      >
+        {description}
+      </Typography>
```

Comment:
```suggestion
      {description ? (
        <Typography
          variant="caption"
          className={clsx(classes.text, classes.description)}
        >
          {description}
        </Typography>
      ) : null}
```
description is also optional.

Reply:
Fixed